src/test/resources/application-test.yaml

```diff
@@ -51,6 +51,8 @@ spring:
     url: iced-latte-minio:9000
     secure: false
     region: us-east-1
+    default-image-directory:
+      products: ./products
```

Comment:
Absence of this setting caused https://github.com/Sunagatov/Iced-Latte/actions/runs/7522255824/job/20474040228?pr=218#step:5:50370